lib/geordi/remote.rb

```diff
@@ -34,18 +35,21 @@ def select_server
     def dump(options = {})
       # Generate dump on the server
       shell options.merge({
-        :remote_command => "dumple #{@config.env} --for_download",
+        :remote_command => "dumple #{@config.env} --for_download && gzip -f #{REMOTE_DUMP_PATH}",
         :select_server => nil # Dump must be generated on the primary server
       })
 
       destination_directory = File.join(@config.root, 'tmp')
       FileUtils.mkdir_p destination_directory
       destination_path = File.join(destination_directory, "#{@stage}.dump")
+      destination_path_gz = "#{destination_path}.gz"
+
       relative_destination = Pathname.new(destination_path).relative_path_from Pathname.new(@config.root)
+      relative_destination_gz = "#{relative_destination}.gz"
 
-      note "Downloading remote dump to #{relative_destination} ..."
+      note "Downloading remote dump to #{relative_destination_gz} ..."
       server = @config.primary_server
-      Util.system! "scp -C #{ @config.user(server) }@#{ server }:#{REMOTE_DUMP_PATH} #{destination_path}"
+      Util.system! "scp -C #{ @config.user(server) }@#{ server }:#{REMOTE_DUMP_PATH_GZ} #{destination_path_gz} && gunzip -f #{destination_path_gz}"
```

Comment:
We already used the `-c` option within scp #52:

From the `scp` manual:

```
-C      Compression enable.  Passes the -C flag to ssh(1) to enable compression.
```

From the `ssh` manual:

```
-C      Requests compression of all data (including stdin, stdout, stderr, and data for
forwarded X11, TCP and UNIX-domain connections).  The compression algorithm is
the same used by gzip(1).  Compression is desirable on modem lines and other slow
connections, but will only slow down things on fast networks.  The default value can be
set on a host-by-host basis in the configuration files; see the Compression option.
```

@triskweline Do you think the manual compression before does provide any advantage?